docs/source/databricks-sql-warehouse.asciidoc

```diff
@@ -71,3 +71,16 @@ where `series name` = ?
 It is important to set `autocommit = true` since the Spark ODBC driver does not support transactions.
 
 Refer to https://learn.microsoft.com/en-us/azure/databricks/integrations/odbc/authentication[Authentication settings for the Databricks ODBC Driver] for alternative authentication options.
+
+Alternatively the `connection_string` can be split and passed as the individual entries:
```

Comment:
I would make this the default and present the connection string as the alternative.